manual/manual/cmds/top.etex

```diff
@@ -1,10 +1,10 @@
-\chapter{The toplevel system (ocaml)} \label{c:camllight}
+\chapter{The toplevel system, aka REPL (ocaml)} \label{c:camllight}
```

Comment:
FWIW, I'd avoid using 'aka' and just use 'or' - the `\pdfchapter` command also should have the change